validator/src/cli.rs

```diff
@@ -612,7 +612,16 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
             Arg::with_name("skip_poh_verify")
                 .long("skip-poh-verify")
                 .takes_value(false)
-                .help("Skip ledger verification at validator bootup"),
+                .help(
```

Comment:
We have a section where we stash deprecated args in validator to kind of get them out of the way:
https://github.com/solana-labs/solana/blob/7d87e8227fde6fb402553e398f5ba7d2bed71ab0/validator/src/cli.rs#L1628

Reply:
Like this https://github.com/solana-labs/solana/pull/30885?